ios/FluentUI.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -6,17 +6,17 @@
         "repositoryURL": "https://github.com/microsoft/appcenter-sdk-apple.git",
         "state": {
           "branch": null,
-          "revision": "25f64229373de97ff3920941cd52203193e5d8be",
```

Comment:
The changes in this file don't seem to be part of the PR.
Do you have your branch up to date with the vnext-tokens branch?

Reply:
Looks like Xcode decided to bump the versions of the demo app's dependencies. Can probably revert this file for now, or maybe just leave it--not sure how/when we update these normally anyway.

Reply:
Yeah I had that question previously to Harrie - what is the standard practice when dealing with automatic changes like this? It seems like they'll have to happen at some point, but I can always revert if that is preferred.